### PR TITLE
CA-173711: Added option to upgrade supp packs during install

### PR DESCRIPTION
**Pull request by koushikcgit**
Adding support to upgrade a supp pack using toolstack plugin
from XenCenter

Signed-off-by: Koushik Chakravarty <koushik.chakravarty@citrix.com>

### CODE REVIEW

**Comment by koushikcgit** [comment]
Adding @simonjbeaumont to review this

**Comment by jonludlam** [comment]
hold on, I'd like to check with srowe whether we can just cause this to be a default.

File: scripts/xe-install-supplemental-pack [hunk 1]
```diff
@@ -11,9 +11,12 @@ Usage:
   exit 1
 }
 
+upgrade=
```
Comment (simonjbeaumont) [inline scripts/xe-install-supplemental-pack]: Is this necessary?

**Comment by johnelse** [comment]
Please make sure the first line of the commit message and pull request title describe the change being made rather than the bug. Further information about the bug can go in the body of the commit message.

File: scripts/plugins/install-supp-pack [hunk 2]
```diff
@@ -28,9 +28,9 @@ def install(session, args):
         session.xenapi.VDI.get_by_uuid(vdi)
     except:
         raise ArgumentError("VDI parameter invalid")
-
+
```
Comment (simonjbeaumont) [inline scripts/plugins/install-supp-pack]: oops.

**Comment by koushikcgit** [comment]
@simonjbeaumont Fixed the extra line. Please have a check now

**Comment by simonjbeaumont** [comment]
Look good now!

**Pull request merged**